### DISCUSSION TITLE
What extension is responsible for this...

### DISCUSSION THREAD

**Discussion by coach77777** [microsoft/vscode-discussions#2057] [Extension Development QnA] [answered]
![Capture](https://github.com/user-attachments/assets/7e46a40a-86a9-4506-bbe4-fdccf216b022)
  
![Capture](https://github.com/user-attachments/assets/d84a38d1-9342-41c6-b951-8d68b1d5b4ef)

yes,
I am having trouble getting rid of the grey definitions in this code. It appeared when I started using Github Copilot. But now I turned Copilot of and am using CodeGeex and the definiton remain. 

I have googled and searched. the problem is I am not sure what these are called but it is making a mess in my code as you can see

**Comment by gjsjohnmurray** [comment, marked as answer]
Does the Start Extension Bisect command help you?
